### DISCUSSION TITLE
Move starters to their own repo

### DISCUSSION THREAD

**Discussion by assapir** [loco-rs/loco#788] [Ideas]
this will make the main repo more focused on one thing and will make contributions easier. It might also be true for site and examples and everything else that isn't the crate itself

**Comment by kaplanelad** [comment]
Keeping the starters as part of the main repository helps us manage versioning, test for breaking changes, and ensure compatibility more effectively.

In our roadmap, we plan to allow pulling starters from remote repositories (similar to a marketplace), but we haven't reached that stage yet.

**Comment by joshka** [comment]
Picking up the "make contributing easier" idea. The obvious thing which makes this mono-repo complex is that there's a lot of folders are the root that are part of the main core crate, and then a bunch which are part of the sub crates. Consider moving the loco-rs crate into a directory to clean this up.

Additionally, while there's a nice STARTERS_LOCAL_PATH environment variable that makes it easy to develop changes to starters, when I was looking to understand how to do this, I found myself trying to look for a way to change the source git repo rather than add an environment variable (and almost started implementing that when I found the override). It may be useful to add a `--templates-repo` to the command line options to make this easier.
